docs/glossary.md

```diff
@@ -3,175 +3,27 @@ id: glossary
-
-- A single Worker actually contains both an [**Activity Worker**](#worker) and a [**Workflow Worker**](#worker), abstracting the logical separation and having the ability to execute both types of tasks.
-- The Worker polls the Temporal service for [**Tasks**](#task), performs those [**Tasks**](#task), and communicates [**Task**](#task) execution results back to the Temporal service.
-- Worker services are developed, deployed, and operated by Temporal customers.
-
-### Workflow
-
-A fault-oblivious stateful function that orchestrates Activities.
-
-- A Workflow has full control over which [**Activities**](#activity) are executed, and in which order.
-- A Workflow must not affect the external world directly, only through [**Activities**](#activity).
-- What makes Workflow code a Workflow is that its state is preserved by Temporal. Therefore any failure of a [**Worker**](#worker) process that hosts the Workflow code does not affect the [**Workflow Execution**](#workflow-execution). The Workflow continues as if these failures did not happen. At the same time, [**Activities**](#activity) can fail any moment for any reason.
-- Because Workflow code is fully fault-oblivious, it is guaranteed to get notifications about [**Activity**](#activity) failures or timeouts and act accordingly.
-- There is no limit to the duration of a Workflow.
-
-### Workflow Execution
-
-An instance of a [**Workflow**](#workflow).
-
-- It is possible for a Workflow Execution to be comprised of multiple [**Workflow**](#workflow) runs. When a [**Workflow's**](#workflow) [**Event History**](#event-history) grows too large it, the next invocation can be called with a "Continue as New" flag to create a new run automatically.
-
-### Workflow Id
-
-A unique identifier for a [**Workflow Execution**](#workflow-execution).
-
-- Temporal guarantees the uniqueness of an Id within a [**Namespace**](#namespace).
-- An attempt to start a [**Workflow**](#workflow) with a duplicate Id results in an **already started** error if there is another open Workflow execution. However, this behavior depends on the `WorkflowIdReusePolicy` flag; If set to `ALLOW_DUPLICATE`, it is possible to start a new execution with the same the same Workflow Id.
-
-### Workflow Task
-
-A Workflow Task is a [**Task**](#task) that contains invocation information for a [**Workfow**](#workflow).
-
-- Every time a new external event that might affect a [**Workflow**](#workflow) state is recorded, a Workflow Task that contains it, is added to a [**Task Queue**](#task-queue) and then picked up by a [**Workflow Worker**](#worker).
-- After the new event is handled, the Workflow Task is completed with a list of [**Commands**](#command).
-- Handling of a Workflow Task is usually very fast and is not related to the duration of operations that the [**Workflow**](#workflow) invokes.
+These terms have specific definitions within the context of the Temporal product.
+
+- [Activity](terminology.md#activity)
```

Comment:
```suggestion
- [Activity](/docs/terminology/#activity)
```

For consistency (future search and replace) lets do relative links like so:
`/docs/<page>/#<ref>`

Reply:
Right-o. Pushing changes in 3...2...1...